libs/SalesforceHybrid/src/com/salesforce/androidsdk/phonegap/plugin/SmartStorePlugin.java

```diff
@@ -59,25 +59,11 @@
 public class SmartStorePlugin extends ForcePlugin {
 
 	// Keys in json from/to javascript
-	public static final String BEGIN_KEY = "beginKey";
```

Comment:
These constants moved to QuerySpec.java (
https://forcedotcom.github.io/SalesforceMobileSDK-Android/com/salesforce/androidsdk/smartstore/store/QuerySpec.html)
 many releases back.